docs/source/technical_tutorials/visualization/stac_ipyleaflet.ipynb

```diff
@@ -1,70 +1,101 @@
 {
```

Comment:
<p>Only thing I would do (which I think we already discussed in Slack) is to change the title to use one #. Everything else looks good to me, though!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Oops, thought I had saved that change 😂